app/views/admin/territories/agents/edit.html.slim

```diff
@@ -10,7 +10,7 @@ h1
       = t(".agent_services_legend")
     = simple_form_for @agent, url: update_services_admin_territory_agent_path(current_territory, @agent) do |f|
       .card-body
-        = f.input :service_ids, collection: current_territory.services,
+        = f.input :service_ids, collection: @available_services,
```

Comment:
on pourrait implémenter ça avec `(current_territory.services + @agent.services).uniq` ou `current_territory.services.union(@agent.services)` pour éviter l'itération dans le controller

Reply:
oui ! merci j’avais fait ça au début mais je voulais aussi améliorer l’affichage et je n’avais pas en tête le label_method j’aurais pu aller lire la doc de simple_form mais j’ai préferré hacker 🤦 merci ! je corrige